docker/usr/local/package/espeak.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Exit immediately on error
+set -e
+
+# Get our method as the name of this script and the url
+METHOD=$(basename "$0")
+URL="$1"
+shift
+
+# Set installation prefix, default to /usr/local unless an external power says otherwise
+PREFIX=${PREFIX:-"/usr/local"}
+
+BUILD_FOLDER="/var/tmp/build"
+
+# Dont pull in the toolchain arguments as they seem to break espeaks build
+# . "${PREFIX}/toolchain.sh"
+
+# Setup the temporary build directories
+mkdir -p "${BUILD_FOLDER}"
+cd "${BUILD_FOLDER}"
+
+# Download the source code
+download-and-extract "${URL}"
+
+echo "Configuring using autotools"
+
+# Find the closest configure file to the root
+AUTOGEN_FILE=$(find -type f -path "*${BUILD_FILE_DIR}/autogen.sh" -printf '%d\t%P\n' | sort -nk1 | cut -f2- | head -n 1)
+cd $(dirname "${AUTOGEN_FILE}")
+
+echo "Configuring using configure file ${AUTOGEN_FILE}"
+
+# We have to run ldconfig as it wants to do things with the libraries
+sudo ldconfig
+
+# Run autogen
+./autogen.sh
+
+# Run configure, No CFlags in here as it seems that espeak hates them so much
+./configure "$@" --prefix="${PREFIX}"
```

Comment:
I vaguely remember something about a data folder for this library. You should check the makefile to ensure it is being set to a nice location (if its still a thing)

Reply:
That's why this is `espeak-ng` not `espeak` :P It's so much easier to build and install. You'll note I've also adapted the install script to install the `/usr/local/share` directory that holds the espeak data folder. Currently it installs a bunch of other nonsense too so i'm not sure if we want to limit it to espeak (or maybe unlimit it to the entire /usr/local)